h5py/h5p.pyx

```diff
@@ -1313,8 +1313,8 @@ cdef class PropFAID(PropInstanceID):
         Set the compatibility level for file format. Legal values are:
 
         - h5f.LIBVER_EARLIEST
-        - h5f.LIBVER_V18 (HDF5 1.10.2 or later)
-        - h5f.LIBVER_V110 (HDF5 1.10.2 or later)
+        - h5f.LIBVER_V18
+        - h5f.LIBVER_V110
         - h5f.LIBVER_V112 (HDF5 1.11.4 or later)
```

Comment:
```suggestion
        - h5f.LIBVER_V112
```

Reply:
These notes are still relevant as long as we support any HDF5 1.10 version, no? I could simplify them to say 1.12 or later, since I assume no-one is now using the 1.11 development series.

Reply:
I guess we could also get rid of the comments altogether & just say that any specific version number won't be available if you're using an older HDF5.

Reply:
oh, 🤦🏻 I missread these comments as 1.10.4.